### DISCUSSION TITLE
(question) Open badge and block protocol?

### DISCUSSION THREAD

**Discussion by ghost** [blockprotocol/blockprotocol#834] [Q&A] [answered]
Hi everyone.

I would like to contribute to block protocol with this new idea: open badge in block protocol.

*But... what is Open Badges?*
Open Badges are created and issued to recognise competencies, skills, achievements or attitudes of their recipients

#### "the problem" 
It is common to develop some tool and be left with doubts if the developer who created this tool has some good intention or if the tool has some strange bug to be corrected

#### "the solution"
1. The solution to this problem is to use a credential in Block Hub, in this way publication in Block Hub can only be allowed when verifying the credential, in this case with open badge.
2. You can gain more block hub space by making more components available or gaining more commitment to the block hub community, having an open badge means that you have gained trust, respect, credibility - of course there may be people who will use the open badge for something bad, some exploit to test attack or vulnerability... but this is not done by most people - for that reason I think that the open badge can be interesting as an extra layer of security when developing components in Block Hub
3. My goal is to make Block Hub a safe, verifiable place... things like Twitter have a blue seal icon, which says that the person posting something is real and not a robot
4. Things like Discourse have plugins with nft for this same purpose: user type checking on the network, I wonder if we could have this same user type checking on the block protocol network

#### question/feedback
*What do you think of this idea?*

**Comment by vilkinsons** [comment, marked as answer]
Hi @codehangen, thanks for your suggestion!

Asa first step, we'll be introducing a "Verified" shield alongside the Þ 0.3 specification launch in late-Feb. Initially, we'll be reviewing popular blocks manually for inclusion in the program (and to some extent figuring out additional criteria for inclusion as we go along, documenting as we progress). The verified shield will at minimum communicate that we have **reviewed a block at least once** with a range of quality and security metrics in mind.

In the future we'll consider substituting this for a more robust set of individual measures covering additional areas such as accessibility, and license permissiveness, as well - with the ultimate aim of providing both humans and bots an ability to quickly determine a block's provenance and trustworthiness upfront.

**Reply by ghost** [reply]
Hi nonparibus.

> Hi @codehangen, thanks for your suggestion!
> Asa first step, we'll be introducing a "Verified" shield alongside the Þ 0.3 specification launch in late-Feb. Initially, we'll be reviewing popular blocks manually for inclusion in the program (and to some extent figuring out additional criteria for inclusion as we go along, documenting as we progress). The verified shield will at minimum communicate that we have reviewed a block at least once with a range of quality and security metrics in mind.
> In the future we'll consider substituting this for a more robust set of individual measures covering additional areas such as accessibility, and license permissiveness, as well - with the ultimate aim of providing both humans and bots an ability to quickly determine a block's provenance and trustworthiness upfront.

thank you for feedback here.

**Comment by vilkinsons** [comment]
Separately, we're considering points-based leaderboard/ranking options that might gamify and incentivize contributions. We're keen this doesn't become too commercial, and are more looking to sites like _Stack Overflow_ for inspiration. Suggestions as to how this might work would be very welcome.

**Reply by ghost** [reply]
Hi nonparibus.

> Separately, we're considering points-based leaderboard/ranking options that might gamify and incentivize contributions. We're keen this doesn't become too commercial, and are more looking to sites like Stack Overflow for inspiration. Suggestions as to how this might work would be very welcome.

thank you for feedback here again. *I'm happy with your answer.*
